partials/about.html

```diff
@@ -14,10 +14,10 @@
         <div class="row">
             <div class="about-section supporting-content">
                 <div class="supporting-content-wrapper">
-                    <div class="supporting-section-body col-xs-12 col-sm-6 col-sm-offset-3">
+                    <div class="col-xs-12 col-sm-6 col-sm-offset-3">
                     	<div class="supporting-section-title">About us</div>
                     	<div class="supporting-section-body">
-                        	We are <a href="https://github.com/bobo333" target="_blank">Steve</a>, Pratik, and <a href="http://www.nathanrkane.com" target="_blank">Nathan</a>, the makers of Plenti. We believe that people lead more fulfilling lives in financial security, when they can choose how they spend their time. We hope Plenti sheds light on how attainable financial freedom is for many working people, and inspires the lifestyle adjustments that bring financial independence more quickly, like smart spending and investing.
+                        	We are <a href="https://github.com/bobo333" target="_blank">Steve</a> and <a href="http://www.nathanrkane.com" target="_blank">Nathan</a>, the makers of Plenti. We believe that people lead more fulfilling lives in financial security, when they can choose how they spend their time. We hope Plenti sheds light on how attainable financial freedom is for many working people, and inspires the lifestyle adjustments that bring financial independence more quickly, like smart spending and investing.
```

Comment:
I have my small piece with the extra page done, but I'm lukewarm on whether
it's really that useful. I can put it up for review but I've been trying to
think of something better. I haven't obviously haha. Feels...I dunno. I
thought about the extra inputs and stuff too... I don't know, something
doesn't quite jive. Seems like the single boxes is solid.

Tempted to write something around accepting 401ks, but not quite sure. Mr
mustache had an interesting view on when to stop contributing because you
have enough for target, that would be pretty nice.
On Jan 29, 2015 8:35 AM, "Nathan Kane" <notifications@github.com> wrote:

> In partials/about.html
> <https://github.com/bobo333/financial_freedom/pull/45#discussion_r23780765>
> :
>
> >                      	<div class="supporting-section-title">About us</div>
> >                      	<div class="supporting-section-body">
> > -                        	We are <a href="https://github.com/bobo333" target="_blank">Steve</a>, Pratik, and <a href="http://www.nathanrkane.com" target="_blank">Nathan</a>, the makers of Plenti. We believe that people lead more fulfilling lives in financial security, when they can choose how they spend their time. We hope Plenti sheds light on how attainable financial freedom is for many working people, and inspires the lifestyle adjustments that bring financial independence more quickly, like smart spending and investing.
> > +                        	We are <a href="https://github.com/bobo333" target="_blank">Steve</a> and <a href="http://www.nathanrkane.com" target="_blank">Nathan</a>, the makers of Plenti. We believe that people lead more fulfilling lives in financial security, when they can choose how they spend their time. We hope Plenti sheds light on how attainable financial freedom is for many working people, and inspires the lifestyle adjustments that bring financial independence more quickly, like smart spending and investing.
>
> Haha. Gotta commit some code if you want to be a creator.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/bobo333/financial_freedom/pull/45/files#r23780765>.
>